### DISCUSSION TITLE
Options for submission artefacts / repositories

### DISCUSSION THREAD

**Discussion by jcendava** [Green-Software-Foundation/hack#143] [Ask Anything]
Please could you confirm the options to link to hackathon artefacts (such as code repository)?

- Is a public repository required, or could we provide access to a private GitLab repository? If the latter is possible, could email addresses be supplied to enable access to be granted for individuals?

Longer term, we may be able to host our contributions on public repo's on our organisation's Github account, but this is not possible with the submissions as they stand.

**Comment by russelltrow** [comment]
Hi @jcendava,

Thanks for reaching out. The rules do state that submissions [must be Open Source](https://github.com/Green-Software-Foundation/hack/wiki/Rules-and-Terms#submission). Is there a way you could publish it?

**Reply by jcendava** [reply]
Completely understand re open-source, wasn't sure if that meant at the point of submission. We will provide a repo with public access on our GitHub.

We had some trouble building and referencing plugins outside of the main IF code-base, which is where things currently run for us. Would that be acceptable for the initial submission? Feel we may not have time to resolve before close.
